src/components/templates/searchBar/index.js

```diff
@@ -0,0 +1,4 @@
+import SearchBar from "./SearchBar";
+
+export { SearchBar };
```

Comment:
export default

Reply:
closing this PR considering the time constraints, will cover this point in future branches